src/components/hv-view/index.js

```diff
@@ -40,6 +40,69 @@ export default class HvView extends PureComponent<HvComponentProps> {
 
   props: HvComponentProps;
 
+  isHiddenElement = (attributes: Attribute[]): boolean => {
+    for (let j = 0; j < attributes.length; j += 1) {
+      if (attributes[j].name === 'hide' && attributes[j].value === 'true') {
+        return true;
+      }
+    }
+    return false;
+  };
+
+  canRenderElement = (element: Element): boolean => {
+    const nonRenderingElements = [
```

Comment:
hard-coding the non-rendering elements here is not very future-proof. If someone adds support for a new non-rendering element, they won't necessarily know to update `HvView`.

A more future-proof approach would be to check for the rendering elements in the component registry. As part of the `options` prop passed to `HvView`, there is an attributed called `componentRegistry` ([link](componentRegistry)). You can look up elements in `componentRegistry` (similar to what we do [here](https://github.com/Instawork/hyperview/blob/master/src/services/render/index.js#L64). I have a future PR that cleans up this access with a helper method).